chpl/chpl-service/src/main/java/gov/healthit/chpl/permissions/domains/activity/GetAnnouncementMetadataActionPermissions.java

```diff
@@ -0,0 +1,20 @@
+package gov.healthit.chpl.permissions.domains.activity;
+
+import org.springframework.stereotype.Component;
+
+import gov.healthit.chpl.permissions.domains.ActionPermissions;
+
+@Component("activityGetAnnouncementMetadataActionPermissions")
+public class GetAnnouncementMetadataActionPermissions extends ActionPermissions {
+
+    @Override
+    public boolean hasAccess() {
+        return getResourcePermissions().isUserRoleAdmin() || getResourcePermissions().isUserRoleOnc();
```

Comment:
How does this work with the rule of anyone can see public announcement activity but only logged in users can see all/private announcement activity?

Reply:
IMO - The permissions will need to be changed when we create the announcements activity report.  For now, the endpoint is only used by the User Actions Activity report, which is only available to ADMIN and ONC.

Reply:
I had already resolved this one! Agree with you!